src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/StreamHelper.cs

```diff
@@ -292,7 +292,7 @@ internal static async Task CopyToAsync(this Stream source, Stream destination, T
             if (perReadTimeout == Timeout.InfiniteTimeSpan)
             {
                 // No timeout - use fast path
-                await source.CopyToAsync(destination, cancellationToken);
+                await source.CopyToAsync(destination, cancellationToken).ConfigureAwait(false);
```

Comment:
Eh, it would be great to cover the code path by test too.

Reply:
> Eh, it would be great to cover the code path by test too.

Honestly the test I did add felt a little arbitrary since it's really any await. I'll be submitting a PR to add the analyzer which will be a little bit better at ensuring the same doesn't happen again